infrastructure/application/services/lps.ts

```diff
@@ -85,6 +116,18 @@ export const createLocalPlanningServices = () => {
     bucket: lpsBucket,
     logsBucket,
     domain,
+    acmCertificateArn: sslCert.arn,
+    mode: "static",
+  });
+
+  new cloudflare.Record("localplanningservices", {
+    // TODO: Update for production!
+    name: "localplanning.editor.planx.dev",
+    type: "CNAME",
+    zoneId: config.require("cloudflare-zone-id"),
+    value: cdn.domainName,
+    ttl: 1,
+    proxied: false, // This was causing infinite HTTPS redirects, so let's just use CloudFront only
```

Comment:
Comment copied across as a reminder to get this issue resolved (or to rely on an AWS service here for WAF etc).